packages/rum-core/src/domain/rumEventsCollection/view/trackReplayStats.ts

```diff
@@ -0,0 +1,20 @@
+import { ViewReplayStats } from '../../../rawRumEvent.types'
+import { LifeCycle, LifeCycleEventType } from '../../lifeCycle'
+
+export function trackReplayStats(lifeCycle: LifeCycle, viewId: string, callback: (stats: ViewReplayStats) => void) {
+  const replayStats: ViewReplayStats = {
+    records_count: 0,
+    segments_count: 0,
+    segments_total_raw_size: 0,
+  }
+
+  const { unsubscribe: stop } = lifeCycle.subscribe(LifeCycleEventType.REPLAY_STATS_UPDATED, (update) => {
```

Comment:
My mental model when reasoning about `trackers` is: 
`tracker` -> notify through `life cycle event or observable` -> `collection`. 
I know there is currently no real rules about it but don't you think it could be interesting to think of it?